### PR TITLE
AzureRT Sprint 45 PR

### PR DESCRIPTION
**Pull request by hyonholee**
1. Add 'ForceUpdate' parameter for the following cmdlets:

Set-AzureVMExtension, 
Set-AzureVMCustomScriptExtension, 
Set-AzureVMAccessExtension
1. Show 'Regions' property for Get-AzureVMAvailableExtensions cmdlet.
2. Add 'ResizedSizeInGB' parameter for the following cmdlets:

Update-AzureDisk
Set-AzureOSDisk

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@hyonholee__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hylee). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@hyonholee Please fix the merge conflicts

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LocationBasedReservedIPTests.cs [hunk 1]
```diff
@@ -62,6 +63,96 @@ public void Cleanup()
...
+        public void CreateReservedIPThenPaaSVM()
+        {
+            try
+            {
+                string reservedIpName1 = "ResrvdIP1";
+                string reservedIpName2 = "ResrvdIP2";
+                string reservedIpLabel1 = Utilities.GetUniqueShortName("ResrvdIPLbl", 5);
+                string reservedIpLabel2 = Utilities.GetUniqueShortName("ResrvdIPLbl", 5);
+                string dnsName = Utilities.GetUniqueShortName("Dns");
+                //string vmName = Utilities.GetUniqueShortName(vmNamePrefix);
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LocationBasedReservedIPTests.cs]: Nit.  Please remove icommented code n next sprint

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LocationBasedReservedIPTests.cs [hunk 2]
```diff
@@ -62,6 +63,96 @@ public void Cleanup()
...
+                string deploymentName = Utilities.GetUniqueShortName("Depl");
+                var input1 = new ReservedIPContext()
+                {
+                    //Address = string.Empty,
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LocationBasedReservedIPTests.cs]: Same.

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/PrivateConfig.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "PrivateConfig": { "Password": "p@ssw0rd" }
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/PrivateConfig.json]: This looks like a credential - I assume this isn't an actual one

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/PublicConfig.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "PublicConfig": { "UserName": "dileep" }
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/PublicConfig.json]: This looks like a real user name, can we substitue a more obviously fake user name?.

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/ServiceConfiguration.cscfg [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-16"?>
+<ServiceConfiguration xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" serviceName="HelloWorld" osFamily="2" osVersion="*" xmlns="http://schemas.microsoft.com/ServiceHosting/2008/10/ServiceConfiguration">
+	<Role name="HelloWorld_WebRole">
+		<ConfigurationSettings>
+			<Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" value="DefaultEndpointsProtocol=https;AccountName=samplesa;AccountKey=1VtQsFrdpGyY8bKCC2WF74B1lV9fBpPtxZbBj6apwFNLCm/sYTlsrCmps9TgBMykw5NiGGGwngYUzhvpECUXUQ==" />
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/ServiceConfiguration.cscfg]: Again, this looks like a real credential - please replace

Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/ServiceConfiguration.cscfg]: Still need to fix this one

**Comment by markcowl** [comment]
@hyonholee Please resolve issues and retarget at the release branch

**Pull request closed without merging**